Explorer/Assets/DCL/UI/Buttons/HoverableButton.cs

```diff
@@ -30,18 +27,21 @@ private void OnDisable()
             Button.onClick.RemoveListener(OnButtonPressed);
         }
 
-        private void OnButtonPressed()
+        public void OnPointerEnter(PointerEventData eventData)
         {
-            UIAudioEventsBus.Instance.SendPlayAudioEvent(ButtonPressedAudio);
-        }
-
-        public void OnPointerEnter(PointerEventData eventData) {
             UIAudioEventsBus.Instance.SendPlayAudioEvent(ButtonHoveredAudio);
             OnButtonHover?.Invoke();
         }
 
         public void OnPointerExit(PointerEventData eventData) =>
             OnButtonUnhover?.Invoke();
 
+        public event Action OnButtonHover;
```

Comment:
Please move upwards to the rest of the field definitions

Reply:
good point! not sure why it was not auto-formatted :hmm